action.yml

```diff
@@ -84,14 +101,14 @@ runs:
          PACKAGES: "${{ inputs.packages }}"
 
     - id: upload-logs
-      if: ${{ inputs.debug == 'true' }}
+      if: env.CACHE_KEY && inputs.debug == 'true'

```

Comment:
Shouldn't we upload the log regardless?

Reply:
Then, we'll get `Warning: No files were found with the provided path: ~/cache-apt-pkgs/*.log. No artifacts will be uploaded` because no log is generated when the package list is empty. Should we suppress this warning, or leave it as is for future changes?